c10/core/impl/alloc_cpu.cpp

```diff
@@ -45,6 +50,35 @@ void memset_junk(void* data, size_t num) {
   }
 }
 
+#ifdef __linux__
```

Comment:
Android is Linux, but we want to hide those in case of an Android, don't we?
```suggestion
#if defined(__linux__) && !defined(__ANDROID__)
```